pkg/reconciler/subscription/subscription_test.go

```diff
@@ -470,9 +470,23 @@ func TestAllCases(t *testing.T) {
 					}}),
 				),
 				// Role
-				CreateRole(channelNS, "test-role", "knsubscribe", "InMemoryChannel", "messaging.knative.dev"),
+				CreateRole(channelNS, "test-role",
+					WithRoleRules(
+						WithPolicyRule(
+							WithAPIGroups([]string{"messaging.knative.dev"}),
+							WithResources("InMemoryChannel"),
+							WithVerbs("knsubscribe")))),
 				// Rolebinding
-				CreateRoleBinding(channelNS, "test-role", NewServiceAccount("test-user")),
+				CreateRoleBinding(channelNS, "test-role",
```

Comment:
we pass channelNs and then "test-role" but the CreateRoleBinding function expected roleName and then namespace

Reply:
Same for `CreateRole`

Reply:
Oh oops I didn't realize, thank you so much!